harvard-lts/CURIOSity
# Fixing style for no-thumb theme.

**Fixing style for no-thumb theme.**
* * *

**JIRA Ticket**: [LTSVIEWER-250](https://at-harvard.atlassian.net/browse/LTSVIEWER-250)

# What does this Pull Request do?
This fixes the problem with the "Open Full Screen Viewer" and "Try new beta Viewer" links on items that have a No Thumb theme in the exhibit.

# How should this be tested?

A description of what steps someone could take to:
* Rebuild CURIOSity using the `LTSVIEWER-250` branch
* Change one of your exhibits to use the "No Thumbnail or Viewer" theme.
* Confirm that the "Open Full Screen Viewer" and "Try new beta Viewer" links do not show up on item pages in that exhibit.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@f8f8ff @enriquediaz
